document/js-api/index.bs

```diff
@@ -1780,7 +1778,7 @@ An implementation must throw a {{RuntimeError}} if one of the following limits i
 In practice, an implementation may run out of resources for valid modules below these limits.
 
 <ul>
-<li>The maximum size of a table is 10,000,000.</li>
+<li>The maximal initial or maximum size of a table is 10,000,000.</li>
```

Comment:
This is incorrect. This is a runtime condition; it's about the actual size of the table, not the limits.